### ISSUE TITLE
June endgame

### ISSUE THREAD

**Issue by roblourens** [Microsoft/vscode#29518]
## Endgame Schedule
- *June 26*, Code freeze for the endgame
- *July 7*, Endgame done

#### Monday, June 26
- [x] Code freeze at 5pm PT
- [x] Ensure we have a green build on all platforms
- [x] All test items contain sufficiently comprehensive test descriptions by 6pm PT

#### Tuesday, June 27
- [x] [Test plan items](https://github.com/Microsoft/vscode/issues?q=is%3Aissue+label%3Atestplan-item+milestone%3A"June+2017") assigned
- [x] [All closed feature-requests either have a verification-needed or on-testplan tag](https://github.com/Microsoft/vscode/issues?q=label%3Afeature-request+-label%3Aon-testplan+-label%3Averification-needed+-label%3Aduplicate+milestone%3A%22June+2017%22+is%3Aclosed)
- [x] Testing
- [x] [Verification needed](https://github.com/Microsoft/vscode/issues?q=is%3Aissue+-label%3Averified+is%3Aclosed+label%3Averification-needed)

#### Wednesday, June 28
- [x] Testing
- [x] Remind team members to assign issues that they intend to fix to the June milestone
- [x] Fixing (self-assigned, milestone assigned)
- [x] [Verification](https://github.com/Microsoft/vscode/issues?q=is%3Aissue+label%3Abug+-label%3Averified+is%3Aclosed+-label%3Aduplicate+-label%3Ainvalid+-label%3Aas-designed+milestone%3A%22June+2017%22)

#### Thursday, June 29
- [x] Fixing last issues
- [x] [Verification](https://github.com/Microsoft/vscode/issues?q=is%3Aissue+label%3Abug+-label%3Averified+is%3Aclosed+-label%3Aduplicate+-label%3Ainvalid+-label%3Aas-designed+milestone%3A%22June+2017%22)
- [x] Run [OSS tool](https://github.com/Microsoft/vscode-distro/blob/master/distro-tools/README.md) after merging shrink-wrap findings **endgame master**
  - *The LCA review of the ThirdPartyNotices.txt files is not needed anymore*
- [x] Check new OSS usage is entered into the [OSS registry](https://ossmsft.visualstudio.com/_apps/hub/ms.vss-oss-web.hub-oss) **endgame master**

#### Friday, June 30
- Satellite modules/npm packages ready, version updated, smoke tested
  - [x] vscode **@bpasero**
  - [x] yo generator **@dbaeumer**
  - [x] vsce **@joaomoreno**
  - [x] node debug **@weinand**
  - [x] node debug2 **@roblourens**
- [x] Check translations (build locally and see the amount of untranslated strings) @michelkaporin
- [ ] All issues [verified](https://github.com/Microsoft/vscode/wiki/Issue-Tracking#verification)
- [x] Fixing (only critical bugs - no string changes)
- [Smoketest](https://github.com/Microsoft/vscode/wiki/Smoke-Test)
  - [x] Windows - @egamma 
  - [x] OS X - @sandy081
  - [x] Linux - @joaomoreno
- [x] All release notes updated - release notes are collected in a file named *`Month_Year.md`* in this [repo directory](https://github.com/Microsoft/vscode-docs/blob/vnext/release-notes/). This includes new features and notable fixes.
  - [x] @alexandrudima
  - [x] @weinand
  - [x] @chrmarti
  - [x] @tyriar
  - [x] @dbaeumer
  - [x] @bpasero
  - [x] @egamma
  - [x] @isidorn
  - [x] @joaomoreno
  - [x] @jrieken
  - [x] @kieferrm
  - [ ] @aeschli
  - [x] @rebornix
  - [x] @ramya-rao-a
  - [x] @roblourens
  - [x] @sandy081
  - [x] @mjbvz
  - [x] @michelkaporin
- [ ] Acknowledge pull requests in release notes. We acknowledge PRs from outside the team. Use the [thankyou](https://vscode-tools.azurewebsites.net/#acknowledgePRs) tool to generate the initial contents of the section. **owner**
  - [x] vscode **endgame master**
  - [x] vscode-node-debug **@weinand**
  - [x] vscode-node-debug2 **@roblourens**
  - [x] vscode-filewatcher-windows **@bpasero**
  - [x] vscode-languageserver-node **@dbaeumer**
  - [ ] language-server-protocol **@dbaeumer**
  - [ ] vscode-textmate **@alexandrudima**
  - [ ] vscode-loader **@alexandrudima**
  - [ ] vscode-generator-code **@aeschli**
  - [x] vscode-vsce **@joaomoreno**
  - [ ] vscode-docs **@gregvanl**
  - [ ] vscode-css-languageservice **@aeschli**
  - [ ] vscode-json-languageservice **@aeschli**
  - [ ] vscode-html-languageservice **@aeschli**
  - [ ] jsonc-parser **@aeschli**
  - [ ] vscode-jscs **@cdias**
  - [x] vscode-tslint **@egamma**
  - [x] vscode-eslint **@dbaeumer**
  - [ ] vscode-jshint **@kieferrm**
  - [x] localization **@danyeh @michelkaporin**
- When done fixing/verifying and there are changes since last build at the end of day PT
  - [x] Trigger new insider build and publish it manually **@owner**

##### Friday/Monday
- [x] Disable continuous `insider` builds - **endgame master**
- [x] Branch code to `release/<x.y>` and release `master` - **endgame master**
- [x] Bump up the version in `package.json` - **endgame master**
- [x] Announce master is open for business **endgame master**
- [x] Polish release notes **@redmond**

##### Tuesday, July 4 - Redmond holiday
- [x] Fireworks
- [x] Grill stuff

##### Monday - Wednesday
- [x] Polish release notes **@redmond**
- [x] Cherry-pick hand-picked and reviewed changes to `release/<x.y>` **@owner**
- [x] Build `Insider` from `release/<x.y>` **endgame master**
- [x] Manually release `Insider` **endgame master**
- [ ] Documentation updated
  - [ ] @alexandrudima
  - [ ] @weinand
  - [ ] @chrmarti
  - [ ] @tyriar
  - [ ] @dbaeumer
  - [ ] @bpasero
  - [ ] @egamma
  - [x] @isidorn
  - [ ] @joaomoreno
  - [x] @jrieken
  - [x] @kieferrm
  - [ ] @aeschli
  - [ ] @rebornix
  - [ ] @ramya-rao-a
  - [x] @roblourens
  - [x] @sandy081
  - [ ] @mjbvz

> **Note:** The `Insiders` build needs to be in the wild for 24 hours before we can enter the last phase of the endgame.

##### Thursday/Friday
- [x] Merge translations **@zurich**
- [x] Build stable for all platforms **endgame master**
- [x] Make rpm signing request @Tyriar
- [x] Sanity check of installable bits
  - [x] Windows
    - [x] signed installer **@owner**
    - [x] zip **@owner**
  - [x] OS X - **@owner**
  - [x] Linux
    - [x] deb package 32-bit **@owner**
    - [x] deb package 64-bit **@owner**
    - [x] rpm package 64-bit **@owner**
    - [x] rpm package 32-bit **@owner**
    - [x] archives **@owner**
- [x] Publish website **@gregvanl**
- [x] Publish to stable **@owner**
- [x] Publish deb and rpms to repositories manually @Tyriar
- [x] Add version with symbols to HockeyApp @bpasero
- [x] Add a git tag to `HEAD` of `release/<x.y>` in format `x.y.z` **endgame master**
- [x] Enable scheduled `insider` builds **endgame master**
- [x] Twitter announcement **@seanmcbreen**

**Comment by roblourens** [comment]
## Releasing 1.14.2

- [ ] Wed pm Redmond: build, validate issues are resolved, smoke test
- [ ] Redmond: Publish insiders with new TS
- [ ] Thu am Zurich: pick up latest bits, validate, smoke test
- [ ] Thu am Redmond: Sync with Zurich, see if any new issue arise
- [ ] Thu pm Redmond: ship 1.14.2

**Comment by joaomoreno** [comment]
@roblourens Should this be closed?
